frontends/PyCDE/src/pycde/module.py

```diff
@@ -204,9 +204,14 @@ def instantiate(self, instance_name: str, inputs: dict, loc):
     if self.extern_name is None:
       return self.circt_mod.create(instance_name, **inputs, loc=loc)
     else:
+      paramdecl_list = [
+          hw.ParamDeclAttr.get(p.name, mlir.ir.TypeAttr.get(p.attr.type),
```

Comment:
Where were we doing this previously?

Reply:
It was in the _hw_ops_ext before... I will move this bit to _msft_ops_ext so we don't have to worry about it everywhere.